### PR TITLE
Update README and contribution guide

### PR DESCRIPTION
**Pull request by gitseti**
- Update `README.md` => see https://github.com/hivemq/mqtt-cli/tree/update-readme 
	- Remove duplicate documentation
	- Add gifs for the main features
- Update `CONTRIBUTING.md` => see https://github.com/hivemq/mqtt-cli/blob/update-readme/CONTRIBUTING.md 
	-  Replace guide with a step by step guide containing images

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,55 +1,123 @@
...
+For some general information on how the HiveMQ Community is organized and how contributions will be accepted please have a look at 
+our [HiveMQ Community Repo](https://github.com/hivemq/hivemq-community). 
 
-Welcome to the HiveMQ Community! Glad to see your interest in contributing to MQTT CLI.
-Please checkout our [Contribution Guide](https://github.com/hivemq/hivemq-community/blob/master/CONTRIBUTING.adoc) to 
-make sure your contribution will be accepted by the HiveMQ team.
+## Prerequisites
+
+We recommend to develop your changes by using [IntelliJ IDEA](https://www.jetbrains.com/idea/). 
+Therefore, the following guide will focus on creating your changes by using IntelliJ.
```
Comment (LukasHiveMQ) [inline CONTRIBUTING.md:10]: I think the "by" can be omitted

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,55 +1,123 @@
...
+- To build mqtt-cli use `./gradlew build`
+- Before proceeding to the next step, please make sure that your changes build and all tests succeed
+- Tip: After building mqtt-cli you can execute the cli by using the generated jar file `java -jar build/libs/mqtt-cli-<version>.jar` (you can also just build this jar by using `./gradlew shadowJar`)
+- Tip: To quickly test your changes manually you can directly execute commands from your IDE by going to <i>Run > Edit Configurations...</i> and adding your cli arguments
+
+![run-config.png](img/run-config.png)
+
+
+## 6. Commit and Push
+- Before committing execute `./gradlew licenseFormat` to add license headers to each file
```
Comment (LukasHiveMQ) [inline CONTRIBUTING.md:82]: Something like:
- Format each file you changed using the .editorconfig settings of the project.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,55 +1,123 @@
...
+- Commit your changes directly from IntelliJ by pressing `Command` + `K` OR via the command `git commit -m "your commit mesage"`
+  - Commits should be as atomic as possible
+  - Commit messages should describe the changes clearly
+  - Commit messages should start with a capital letter for consistency
```
Comment (LukasHiveMQ) [inline CONTRIBUTING.md:86]: - Commit messages should be in an imperative simple present form

File: README.md [hunk 4]
```diff
@@ -1,227 +1,68 @@
...
-- Configurable default settings
-- Ability to connect various MQTT Clients to different brokers simultaneously
-- Quick broker tests
-- Export information from HiveMQ API endpoints
-- Various distributions available
+## Getting Started
 
-## Prerequisites
+- [Installation instructions](https://hivemq.github.io/mqtt-cli/docs/installation/) 
+- [Full documentation](https://hivemq.github.io/mqtt-cli)
```
Comment (LukasHiveMQ) [inline README.md:29]: - For a development setup please have a look at the [Contributing.md] (...)

File: README.md [hunk 5]
```diff
@@ -1,227 +1,68 @@
...
 
-## Contributing
 
-See [CONTRIBUTING.md](CONTRIBUTING.md)
+<img align="center" src="./img/test.gif"/>
 
-## License
 
-See [LICENSE](LICENSE)
+## How to contribute
```
Comment (LukasHiveMQ) [inline README.md:66]: How to Contribute

File: README.md [hunk 6]
```diff
@@ -1,227 +1,68 @@
...
 
-# MQTT Command Line Interface
+<a href="https://hivemq.github.io/mqtt-cli/">
+<img src="./img/mqtt-logo.svg" width="500"/>
+</a>
 
 [![GitHub Release](https://img.shields.io/github/v/release/hivemq/mqtt-cli?style=for-the-badge)](https://github.com/hivemq/mqtt-cli/releases)
 [![GitHub Workflow Status (branch)](https://img.shields.io/github/actions/workflow/status/hivemq/mqtt-cli/.github/workflows/check.yml?branch=master&style=for-the-badge)](https://github.com/hivemq/mqtt-cli/actions/workflows/check.yml?query=branch%3Amaster)
 [![CLI Downloads](https://img.shields.io/github/downloads/hivemq/mqtt-cli/total?style=for-the-badge)](https://github.com/hivemq/mqtt-cli/releases)
-[![CLI License](https://img.shields.io/github/license/hivemq/mqtt-cli?style=for-the-badge)](https://github.com/hivemq/mqtt-cli/blob/develop/LICENSE)
```
Comment (LukasHiveMQ) [inline README.md:12]: Would leave in for alignment

File: README.md [hunk 7]
```diff
@@ -1,227 +1,68 @@
...
+<img align="center" src="./img/pubsub.gif"/>
 
-## Documentation
+<br></br>
+[Getting Started](#getting-started) •
+[Publish](#publish) •
+[Subscribe](#subscribe) •
+[Shell](#shell) •
+[Test](#test) •
+[How to contribute](#how-to-contribute)
```
Comment (LukasHiveMQ) [inline README.md:21]: How to Contribute

File: README.md (hunk 4 above)
Comment (LukasHiveMQ) [reply README.md:29]: I would delete the build from source part inside the documentation.

File: README.md [hunk 8]
```diff
@@ -1,227 +1,68 @@
...
-## Contributing
 
-See [CONTRIBUTING.md](CONTRIBUTING.md)
+<img align="center" src="./img/test.gif"/>
 
-## License
 
-See [LICENSE](LICENSE)
+## How to contribute
+- If you want to request a feature or report a bug, please [create a GitHub Issue using one of the provided templates](https://github.com/hivemq/mqtt-cli/issues/new/choose)
```
Comment (LukasHiveMQ) [inline README.md:67]: issue

**Review by LukasHiveMQ — APPROVED** [review]

File: README.md (hunk 8 above)
Comment (gitseti) [reply README.md:67]: GitHub Issue is usually written in upper case.

**Review by SgtSilvio — APPROVED** [review]

**Pull request merged**